<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialPage</class>
 <widget class="QWidget" name="SocialPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>630</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Bitcointalk">
    <attribute name="title">
     <string>          Bitcointalk          </string>
    </attribute>
    <widget class="QWebView" name="webView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>725</width>
       <height>620</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="CoinMarketCap">
    <attribute name="title">
     <string>          Coinmarketcap          </string>
    </attribute>
    <widget class="QWebView" name="webView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>725</width>
       <height>620</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Twitter">
    <attribute name="title">
     <string>          Twitter          </string>
    </attribute>
    <widget class="QWebView" name="webView_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>725</width>
       <height>620</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Reddit">
    <attribute name="title">
     <string>          Reddit          </string>
    </attribute>
    <widget class="QWebView" name="webView_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>725</width>
       <height>620</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
